orcid-core/src/main/java/org/orcid/pojo/ajaxForm/Name.java

```diff
@@ -0,0 +1,72 @@
+package org.orcid.pojo.ajaxForm;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.orcid.core.security.visibility.OrcidVisibilityDefaults;
+
+public class Name extends VisibilityForm implements ErrorsInterface, Serializable{
```

Comment:
@leomendoza123 you dont need this new class, please reuse https://github.com/ORCID/ORCID-Source/blob/master/orcid-core/src/main/java/org/orcid/pojo/ajaxForm/NamesForm.java or let us know why you cant reuse that one.

Thanks

Reply:
Hey @lizkrznarich @leomendoza123, yea, so, instead of:

private Name effective = null;
private Name real = null;  

In the `Names` class, use :
private NamesForm effective = null;
private NamesForm.java real = null;

So, you dont need the Name.java class, you just need to reuse NamesForm.java 

Thanks